.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
remember to add this to the gitignore :)

Reply:
this should be in there in one of the commits

Reply:
wait lol